### PR TITLE
fix: Fix types

### PR DESCRIPTION
**Pull request by segevfiner**

### CODE REVIEW

File: src/components/Typography/SwText.vue [hunk 1]
```diff
@@ -44,14 +38,98 @@ const fontFamily = computed<string>(() => {
...
+const SwTextFontFamilyValues = [
+  'fontfamily-main',
+  'fontfamily-secondary',
+] as const;
+export type SwTextFontFamily = typeof SwTextFontFamilyValues[number];
+
+const SwTextWeightValues = ['light', 'regular', 'bold', 'bolder'] as const;
+export type SwTextWeight = typeof SwTextWeightValues[number];
+
+const textWeights: Record<SwTextWeight, number> = {
```
Comment (yhamry) [inline src/components/Typography/SwText.vue:77]: ❤️

**Review by yhamry — COMMENTED** [review]
Amazing!

**Review by yhamry — APPROVED** [review]
Amazing!

**Comment by segevfiner** [comment]
@talkor ?

**Review by talkor — APPROVED** [review]

**Pull request merged**